src/core/linalg/src/sparse/4C_linalg_sparsematrix.hpp

```diff
@@ -518,24 +518,24 @@ namespace Core::LinAlg
     //@{
 
     /// Returns the result of a matrix multiplied by a Core::LinAlg::Vector<double> x in y.
-    int multiply(
+    void multiply(
         bool TransA, const Core::LinAlg::Vector<double>& x, Core::LinAlg::Vector<double>& y) const;
 
     /// Returns the result of a Epetra_CrsMatrix multiplied by a Epetra_MultiVector X in Y.
     int multiply(bool TransA, const Core::LinAlg::MultiVector<double>& X,
         Core::LinAlg::MultiVector<double>& Y) const override;
```

Comment:
What about this one? It is still returning `int`.

Reply:
Ok, then this needs to be deferred until changing the internals to Tpetra.

Reply:
That's one of the reasons why we want to get rid of the `Epetra_Operator` inheritance.